src/libexpr/flake/flakeref.cc

```diff
@@ -122,6 +122,28 @@ std::pair<FlakeRef, std::string> parseFlakeRefWithFragment(
 
             if (isFlake) {
 
+                if (!allowMissing && !pathExists(path + "/flake.nix")){
+                    notice("path '%s' does not contain a 'flake.nix', searching up",path);
```

Comment:
I’m not entirely convinced by this message. But we’ll see at usage whether it needs to be dropped or not